### ISSUE TITLE
Javascript code snippets not formatting in electron window

### ISSUE THREAD

**Issue by shanel262** [nearform/deck#81]
Snippets of js code do not format correctly in the electron window after the first snippet, however it works in the browser.

![screen shot 2016-06-24 at 10 32 31](https://cloud.githubusercontent.com/assets/10059043/16334113/0a1c5aca-39f7-11e6-8da9-5aab6f3a54da.png)
--The first javascript snippet

![screen shot 2016-06-24 at 10 32 47](https://cloud.githubusercontent.com/assets/10059043/16334128/1e4bee48-39f7-11e6-8638-a4c9c24bedb7.png)
--The second js snippet

**Comment by davidmarkclements** [comment]
so if you do tag the code fence with `js` it doesn't, but if you tag it with `javascript` it does (for example see https://gist.github.com/davidmarkclements/90facaa98ba2126d0b0c4a2d0fa96b01),

 not an electron problem its a problem with the syntax highlighter deck uses (prism) - needs to be addressed in the src/main.js of deck presenter

**Comment by shanel262** [comment]
Yeah I tagged all the snippets with 'javascript' but it still happens. It appears to only be a problem caused by the javascript tag, bash and sh work fine.

**Comment by davidmarkclements** [comment]
can you link a gist of the entire deck.md file

**Comment by shanel262** [comment]
Yup https://gist.github.com/shanel262/21615aa44e4f62a81399c07e498e44a4
